docs/vo_service_discovery/vo_service_discovery.rst

```diff
@@ -0,0 +1,183 @@
+to query this directory by service type, keywords, wavelength band or coverage,
+and source or publisher.
+
+Here is an example showing show to find information about conesearch services
+provided by the NASA Extragalactic Database (NED).
+
+.. code-block:: python
+
+    >>> from astroquery.vo_service_discovery import Registry
+    >>> results = Registry.query(source='ned', service_type='cone')
+    >>> print(results)
+
+                                waveband                         ... service_type
+        -------------------------------------------------------- ... ------------
+                                                        infrared ...   conesearch
+        radio#millimeter#infrared#optical#uv#euv#x-ray#gamma-ray ...   conesearch
+
+The result of the query is an `astropy.table.Table` object. The methods of `Table`
+can be used to inspect the search result.
+
+.. code-block:: python
+    >>> print(results.colnames)
+
+        ['waveband', 'short_name', 'ivoid', 'res_description', 'access_url',
+         'reference_url', 'publisher', 'service_type']
+
+The value of `'access_url'` specifies the endpoint for the service.
+
+Documentation of the service may be found in the `'reference_url'` values.
+We could use capabilities in the standard library to open the first url
+in a web browser.
+
+.. code-block:: python
+
+    >>> import webbrowser
+    >>> webbrowser.open(results['reference_url'][0])
+
+
+Discovering Services
+====================
+
+The `astroquery.vo_service_discovery.Registry.query` method provides several
+ways to search for specific services of interest. Each of these options
+can be combined into a single call to the `query` method.
+
+The ``service_type`` parameter can be set to the VO service types ``'conesearch'``,
+``'simpleimageaccess'``, ``'simplespectralaccess'``, or ``'tableaccess'``. These
+values may be shortened to ``'cone'``, ``'image'``, ``'spectra'`` or
+``'spectrum'``, or ``'tap'`` or ``'table'``.
+
+The value of the ``keyword`` parameter will return results containg that string
+in the ``ivoid``, ``short_name``, or ``res_description`` fields.
+
+Wavelength range can be specified by a string containing one or more band names.
+The options include gamma-ray (wavelength below 0.1 Angstrom), x-ray
+(0.1 to 100 Angstrom), euv (100-1000 Angstrom), uv (1000 to 3000 Angstrom),
+optical (3000 to 10000 Angstrom), infrared (1 to 100 micron), millimeter (0.1 to
+10 mm), or radio (greater than 10 mm wavelength). Multiple bands can be specified
+with a comma-separated list.
+
+The ``source`` parameter is a string that is matched to any substring in ivoid.
+
+The ``publisher`` is the name of any publishing organization.
+
+``order_by`` can be specified to sort the result by any of the column names.
+
+The ``logic_string`` is any other string that is needed for the ADQL ``where``
+clause, and must start with the value ``' and '``.
+
+
+The ADQL query can be printed by specifying ``verbose=True``.
+
```

Comment:
Add to this the other options of `query`.

```suggestion

The results of the query can be saved to a file. Set ``dump_to_file=True`` to save the results table. The format will be ``'votable'`` by default; you can change this with the ``output_format`` option. The result will be saved to a filename using the TAP job ID; you can specify the filename with the ``output_file`` option.

By default, the query is made using the  registry TAP URL. The ``url`` option will override that value. Alternatively, the configuration can be changed to use a different URL.
```

Reply:
Addressed.